data/human/news.txt

```diff
@@ -1199,6 +1294,29 @@ news "human farming in pirate space"
 
 
 
+news "human republic reporter in pirate space"
+	location
+		government "Pirate"
+	name
+		word
+			"Reporter"
+	message
+		word
+			`"`
+		word
+			"I should never have taken this assignment. Yesterday I interviewed a young pirate and he tried to rob me!"
+			"It really isn't so bad here, but I have to make it sound worse than it is to keep my ratings high."
+			"The political hierarchy out here is way more chaotic than any other planet I've visited. In the past year I've had to report on five coups d'etat!"
```

Comment:
```suggestion
			"The political hierarchy out here is way more chaotic than any other planet I've visited. In the past year, I've had to report on five coups d'etat!"
```
Any introductory phrase like this _technically_ needs a comma, but since we're using dialogue, I think it's more appropriate to add commas only when it feels like they're needed. This is a case where I could go either way, just because the sentence feels pretty long without one. Up to you!

Reply:
I agree that it could go either way, but I've decided to commit your suggestion. Thanks for the input!